test/AdobeState.test.js

```diff
@@ -376,7 +379,19 @@ describe('private methods', () => {
       const store = await AdobeState.init(fakeCredentials)
 
       const url = store.createRequestUrl(key, queryParams)
-      expect(url).toEqual(`${myConstants.ADOBE_STATE_STORE_ENDPOINT[env]}/v1/containers/${fakeCredentials.namespace}/data/${key}?${querystring.stringify(queryParams)}`)
+      expect(url).toEqual(`https://${DEFAULT_REGION}.${myConstants.ADOBE_STATE_STORE_ENDPOINT[env]}/${API_VERSION}/containers/${fakeCredentials.namespace}/data/${key}?${querystring.stringify(queryParams)}`)
+    })
+
+    test('no params, region set', async () => {
+      const region = 'apac'
+      const env = PROD_ENV
+      getCliEnv.mockReturnValue(env)
+
+      // need to instantiate a new store, when env changes
+      const store = await AdobeState.init({ ...fakeCredentials, region })
+
+      const url = store.createRequestUrl()
+      expect(url).toEqual(`https://${region}.${myConstants.ADOBE_STATE_STORE_ENDPOINT[env]}/${API_VERSION}/containers/${fakeCredentials.namespace}`)
     })
```

Comment:
missing a test for invalid/not-supported region?

Reply:
fixed via https://github.com/adobe/aio-lib-state/pull/138/commits/06e4ba3ba421260f1d2964b2d98e06ff52c715c0